yourgov/components/StaffMembersTable/DataTable.tsx

```diff
@@ -0,0 +1,388 @@
+											</TableCell>
+										</TableRow>
+									))}
+								</Fragment>
+							) : (
+								<Fragment>
+									{data.map((item) => {
+										const {
+											index,
+											name,
+											role,
+											status,
+											lastActive,
+											dateJoined,
+										} = item;
+
+										// Adding 2 because the table header row is the first row
+										const rowIndex = index + 2;
+										const id = name.replace(' ', '-');
+
+										return (
+											<DataTableRow
+												key={name}
+												selectable={selectable}
+												itemId={id}
+												name={name}
+												rowIndex={rowIndex}
+											>
+												<TableCell as="th" scope="row">
+													<TextLink href={`#${id}`}>{name}</TextLink>
+												</TableCell>
+
+												{/* <DataTableRowAssignee assignee={assignee} /> */}
+												<TableCell>{role}</TableCell>
+
+												<DataTableRowStatus status={status} />
+
+												<TableCell>
+													{lastActive &&
+														`${formatDistance(
+															new Date(LAST_ACTIVE_RELATIVE_TIME_MS),
+															new Date(lastActive)
+														)} ago`}
+												</TableCell>
+
+												<TableCell>
+													{dateJoined &&
+														format(new Date(dateJoined), 'd MMMM yyyy')}
+												</TableCell>
+
+												<TableCell>
+													<DropdownMenu>
+														<DropdownMenuButton>
+															Actions
+															<VisuallyHidden> for {name}</VisuallyHidden>
+														</DropdownMenuButton>
+
+														<DropdownMenuPanel>
+															<DropdownMenuItem onClick={onClickChangeRoles}>
+																Change roles
+																<Text css={visuallyHiddenStyles}>
+																	{' '}
+																	for {name}
+																</Text>
+															</DropdownMenuItem>
+
+															<DropdownMenuItem onClick={onClickPauseAccess}>
+																Pause access
+																<Text css={visuallyHiddenStyles}>
+																	{' '}
+																	for {name}
+																</Text>
+															</DropdownMenuItem>
+
+															<DropdownMenuItem onClick={onClickRemoveAccess}>
+																Remove access
+																<Text css={visuallyHiddenStyles}>
+																	{' '}
+																	for {name}
+																</Text>
+															</DropdownMenuItem>
+														</DropdownMenuPanel>
+													</DropdownMenu>
+												</TableCell>
+
+												<ModalConfirmChangeRole
+													currentRole={item.role}
+													isOpen={modalChangeRoleOpen}
+													onClose={() => setModalChangeRoleOpen(false)}
+													onConfirm={onConfirmChangeRole}
+												/>
+
+												<ModalConfirmPauseAccess
+													itemsToPause={item}
+													isOpen={pauseModalOpen}
+													onClose={() => setPauseModalOpen(false)}
+													onConfirm={onConfirmPause}
+												/>
+
+												<ModalConfirmRemoveAccess
```

Comment:
This seems to render twice 
<img width="790" alt="image" src="https://github.com/user-attachments/assets/4b5bd425-0ae9-4c2c-b29b-7d4cee849356">